doc/sphinx/source/inspection/index.rst

```diff
@@ -0,0 +1,30 @@
+.. inspection/index: 
+
+Visualization Tools
+###################
+
+One option for visualizing the graph or node structure of a graph is to compile 
+with the :envvar:`NGRAPH_ENABLE_VISUALIZE_TRACING` flag. This generates a PDF, 
+though it is not ideal for viewing large graphs.  
+
+To inspect larger graphs, it may be necessary to convert ``.json`` files 
```

Comment:
When visualization is enabled a `dot` file is generated along with the png file. This dot file can be imported into these tools.

Reply:
PR #2821 has some changes that affect how this works, so I tried to add those changes (since they've already been merged).  

> We need to add a new flag to visualize the source graph. 

When it comes to explaining that primitive formats (`.pdf` / `.json` ) are terrible for viewing large graphs, it's probably better to not get into anything too specific until this gets more design work behind it.